templates/firefox-ios.xliff

```diff
@@ -1496,6 +1500,22 @@
         <source>Last week</source>
         <note>History tableview section header</note>
       </trans-unit>
+      <trans-unit id="LibraryPanel.History.ClearGroupedTabsCancel.v100" xml:space="preserve">
+        <source>Cancel</source>
+        <note>Within the History Panel, users can delete search group sites history. They can cancel this action by pressing a cancel button.</note>
+      </trans-unit>
+      <trans-unit id="LibraryPanel.History.ClearGroupedTabsDelete.v100" xml:space="preserve">
+        <source>Delete</source>
+        <note>Within the History Panel, users can delete search group sites history. They need to confirm the action by pressing the delete button.</note>
+      </trans-unit>
+      <trans-unit id="LibraryPanel.History.ClearGroupedTabsTitle.v100" xml:space="preserve">
+        <source>Delete all sites in %@?</source>
+        <note>Within the History Panel, users can delete search group sites history. %@ represents the search group name.</note>
+      </trans-unit>
+      <trans-unit id="LibraryPanel.History.ClearHistoryMenuTitle.v100" xml:space="preserve">
+        <source>Removes history (including history synced from other devices), cookies and other browsing data.</source>
```

Comment:
Just want to confirm this is the actual string for "LibraryPanel.History.ClearHistoryMenuTitle.v100", as it sounds more like a localization comment

Reply:
I got this string from https://mozilla-hub.atlassian.net/browse/FXIOS-3765
So it says the full string is: `"Removes history (including history synced from other devices), cookies and other browsing data."`